changelog/fix_autogenerateconfig_command_ignores_config.md

```diff
@@ -0,0 +1 @@
+* [#10657](https://github.com/rubocop/rubocop/issues/10657): Fix `AutoGenerateConfig` command ignores specified config file by option. ([@nobuyo][])
```

Comment:
Perhaps user knows command line option name, not internal class name, right?

```suggestion
* [#10657](https://github.com/rubocop/rubocop/issues/10657): Fix `--auto-gen-config` command option ignores specified config file by option. ([@nobuyo][])
```

Reply:
Yeah, you're right. I've updated branch.